doc/api/zlib.md

```diff
@@ -505,45 +505,45 @@ that effectively uses an 8-bit window only.
 added: v0.5.8
 -->
 
-* `options` {Object}
+* `options` {zlib.Options}
```

Comment:
Hmm, this makes me think that `zlib.Options` is a constructor I have to instantiate instead of a plain object. I have to click the link to understand that in reality is a plain object.

Reply:
Commit amended. Would it be better in this way?

Reply:
Yes, I think it's better. Thanks.

Reply:
We have something similar for `http2`:

https://github.com/vsemozhetbyt/node/blob/7938680ebb3e1b3319e274c864d71b0b593a862c/tools/doc/type-parser.js#L74-L75